### PR TITLE
final work (rmd, code and figures)

### PR DESCRIPTION
**Pull request by pamela-tueam**
Final work
Rmd file
bib file
python code
figures

### CODE REVIEW

File: 05-nao.Rmd [hunk 1]
```diff
@@ -1,18 +1,135 @@
...
+
+
+```{r eof, fig.cap = "Leading empirical orthogonal function (EOF 1) of the seasonal mean sea level pressure anomalies in the North Atlantic sector (20°–70°N, 90°W–40°E), and the percentage of the total variance it explains. The data cover the period of 1940–2024.", echo=FALSE, warning=FALSE, error=FALSE, message=FALSE, out.width="400px", fig.align="center"}
```
Comment (henrifnk) [inline 05-nao.Rmd:63]: Add your name to the chunk names headers to avoid duplicated chunks.

File: book.bib [hunk 2]
```diff
@@ -191,3 +191,140 @@ @article{zscheischler
...
+  author={Hurrell, James W and Van Loon, Harry},
+  journal={Climatic change},
+  volume={36},
+  number={3},
+  pages={301--326},
+  year={1997},
+  publisher={Springer}
+}
+
+@article{ottersen,
```
Comment (henrifnk) [inline book.bib:216]: Add years to all of your author fields please

**Review by henrifnk — CHANGES_REQUESTED** [review]
LGTM, only Small changes (see in code)

File: 05-nao.Rmd [hunk 3]
```diff
@@ -1,18 +1,135 @@
...
+As implications, regions with strong positive correlations experience atmospheric conditions similar to Iceland (the center of action). These areas may share similar temperature anomalies. For instance, if the region of the center of action experiences warmer-than-average temperatures during high NAO phase, positive correlated regions might also experience similar warmth temperatures. We will also have similar precipitation patterns (if it is drier than the nearby regions may also be similar dry). Conversely, the region with negative correlations such as the anticorrelation center at Azores, will exhibit opposite conditions, including temperature contrasts (warmer vs. cooler) and differing precipitation patterns (wetter vs. drier).
+
+
+```{r correlation, fig.cap = "One-point correlation map of 500 hPa geopotential height for boreal winter (December-March) over 1940-2024. The reference point is 65°N, 18°W. Negative correlation coefficients are in blue and the lines are dashed.", echo=FALSE, warning=FALSE, error=FALSE, message=FALSE, out.width="350px", fig.align="center"}
```
[the remark this answers is not in the document]
Comment (pamela-tueam) [reply 05-nao.Rmd]: done

File: 05-nao.Rmd (hunk 1 above)
Comment (pamela-tueam) [reply 05-nao.Rmd]: done

File: book.bib (hunk 2 above)
Comment (pamela-tueam) [reply book.bib]: done

**Pull request merged**